quinn-proto/src/connection/streams.rs

```diff
@@ -1218,6 +1223,13 @@ pub enum StreamEvent {
         /// Error code supplied by the peer if the stream was stopped
         stop_reason: Option<VarInt>,
     },
+    /// A STOP_SENDING have been received for an outgoing stream
+    Stopped {
+        /// Which stream has been stopped
+        id: StreamId,
+        /// Error code supplied by the peer
+        stop_reason: VarInt,
```

Comment:
```suggestion
        error_code: VarInt,
```
Let's be consistent with the naming in the spec, and the argument naming for `Stream::stop`.